Widget as WC plugin in ui-card

TomaszLukawskiSam opened the pull request:
[Issue] https://github.com/Samsung/TAU/issues/1360
[Problem] HomeApp: change emebeded content to remote web clip
[Solution]
   - new widget - Card
   - examples

singa2000 commented on examples/mobile/HomeApp/webclip/restaurant/hotel-breakfast-wc.js:
```diff
@@ -1,11 +1,184 @@
-/* global tau */
+const tmpl = document.createElement("template");
+
+tmpl.innerHTML = `
```
I just have a question.
There are the css and html code in the js file.
A general developer who distributes web clips, he have to include the css/html code into js fie??
Or it's just temporary?

lslachciak replied:
how about separating them in future like in
https://stackoverflow.com/questions/55080103/how-to-separate-web-components-to-individual-files-and-load-them

TomaszLukawskiSam replied:
This commit is early draft. I try verify different ways to achieving the goal.

TomaszLukawskiSam replied:
Developer who distribute web clip can do as he needs. In my case I wanted to add template to js file. This plugin has been realized by Custom Element but other Web Clips as "TV Remote Control" has been made as html and css.

At the moment plugin need `html` file eg. `index.html` (example: `examples/mobile/HomeApp/webclip/apps-on-tv/index.html`) with
`<div class="ui-card"></div>` element.

TomaszLukawskiSam committed:
(mobile) Card: new widget and examples

[Issue] https://github.com/Samsung/TAU/issues/1360
[Problem] HomeApp: change emebeded content to remote web clip
[Solution]
 - new widget - Card
 - examples

Signed-off-by: Tomasz Lukawski <t.lukawski@samsung.com>

lslachciak commented on examples/mobile/HomeApp/css/style.css:
```diff
@@ -130,26 +24,6 @@
-    align-self: center;
-}
-
 .app-dummy-payment input {
```
this is part of widget too I think

TomaszLukawskiSam replied:
This is little complicated now. The Web Clip will open instant app as full page. This style is required by other page of Home App.

lslachciak commented on examples/mobile/HomeApp/index.html:
```diff
@@ -34,256 +34,15 @@
-				</a>
-			</div>
+			<!-- Home App Plugins -->
+			<div class="ui-card" rel="card" data-src="webclip/weather/index.html"></div>
```
rel attribute is precisely described by https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel
how about using another attribute?

TomaszLukawskiSam replied:
I removed this attribute. Isn't required.
